modules/mod_ginger_rdf/tests/mod_ginger_rdf_tests.erl

```diff
@@ -156,14 +156,26 @@ deserialize_test() ->
 
 rdf_export_test() ->
     Context = context(),
+    %% Predicate must have a URI to be included in export.
+    m_rsc:update(author, [{uri, <<"http://schema.org/author">>}], z_acl:sudo(Context)),
+
     Props = [
         {category, text},
         {title, {trans, [
             {nl, <<"Een mooie titel">>}
         ]}},
         {is_published, true}
     ],
+    AuthorProps = [
+        {category, person},
+        {title, {trans, [
+            {nl, <<"Auteur van het artikel">>}
+        ]}},
```

Comment:
Reminded by that a name is normally not translated (but that is not the point here I know) I am wondering if it would be good to also check what happens with untranslated values?

Reply:
Good idea! Made the property monolingual so we’re now testing the two variants.